project2/quick.py

```diff
@@ -8,6 +8,6 @@ def get_trending(api_key):
     return data
 
 if __name__ == "__main__":
-    api_key = "ZYdJkh7f8h7myXG5gfjUd3jZT1BoshEo"
+    api_key = "1rcKChP84QC5RgzDuP3UyFzz79uPXwym"
```

Comment:
You'll want to remove this from your git history, since it's like a password. There are instructions on how to do this [here](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/removing-sensitive-data-from-a-repository).

Reply:
oh crap, I did not even realize I left it there. Thank you!